### ISSUE TITLE
Hover Effect on MENU ITEMS should be present to increase Interactivity with user

### ISSUE THREAD

**Issue by SurbhiRathi23** [sairish2001/makesmatheasy.github.io#42]
Here hover effect should be present+ font styles can be changed 

![WhatsApp Image 2021-03-09 at 3 44 43 PM](https://user-images.githubusercontent.com/55491101/110455631-ac868180-80ee-11eb-96d4-12b99b8c49b0.jpeg)

**Comment by SurbhiRathi23** [comment]
Hello sir, I want to work on this issue and want to contribute to this. I am a participant of GSSOC 2021. 
Could you please assign me this issue as a part of GSSOC 2021.

**Comment by SurbhiRathi23** [comment]
@sairish2001  sir please assign it to me.

**Comment by sairish2001** [comment]
@SurbhiRathi23 Go ahead, and feel free to ask your doubts,if any

**Comment by sairish2001** [comment]
But, hover effect does not work properly in  mobile devices, do apply hover effects only or desktop view, it should not be visible on mobile devices

**Comment by SurbhiRathi23** [comment]
okay sir

**Comment by SurbhiRathi23** [comment]
@sairish2001  I have added hover effects and it is also working fine on mobile devices. If you want, I can keep it only for large screen size. 

I have also added some code to change the active button on navigating from one to other page using JavaScript.

![c1](https://user-images.githubusercontent.com/55491101/110591675-5f161d00-819f-11eb-9f4b-eec9be661aa3.JPG)
![c3](https://user-images.githubusercontent.com/55491101/110591680-61787700-819f-11eb-9259-8bf7be20057e.JPG)
![c2](https://user-images.githubusercontent.com/55491101/110591689-63423a80-819f-11eb-9645-308bba9d8b74.JPG)

Should I make pull request?
